Unix/configeditor/configeditor.cpp

```diff
@@ -44,6 +44,7 @@ OPTIONS:\n\
     -s, --set VALUE             Set property name to specific value.\n\
     -c, --comment               Comment out property name specified.\n\
     -u, --uncomment             Uncomment out property name specified.\n\
+    -q, --query                 Check if property contains queried value.\n\
```

Comment:
to be consistent with previous usage, may below you'd better use below for this:
```
-q, --query VALUE             Check if property contains queried value.\n\ 
```